test/stripe/stripe_client_test.rb

```diff
@@ -143,14 +143,19 @@ class StripeClientTest < Test::Unit::TestCase
 
       context "Stripe-Account header" do
         should "use a globally set header" do
-          Stripe.stripe_account = 'acct_1234'
+          begin
+            old = Stripe.stripe_account
+            Stripe.stripe_account = 'acct_1234'
 
-          stub_request(:post, "#{Stripe.api_base}/v1/account").
-            with(headers: {"Stripe-Account" => Stripe.stripe_account}).
-            to_return(body: JSON.generate(API_FIXTURES.fetch(:account)))
+            stub_request(:post, "#{Stripe.api_base}/v1/account").
+              with(headers: {"Stripe-Account" => Stripe.stripe_account}).
+              to_return(body: JSON.generate(API_FIXTURES.fetch(:account)))
 
-          client = StripeClient.new
-          client.execute_request(:post, '/v1/account')
+            client = StripeClient.new
+            client.execute_request(:post, '/v1/account')
+          ensure
+            Stripe.stripe_account = old
+          end
```

Comment:
This block is unrelated to my change, but happened to notice that this global state is never unset in this test which is poor hygiene. I put it in here for convenience.